<?xml version="1.0"?>
<!--XML-->
<launch>

     <!-- SETUP ARGUMENTS -->
    <!-- enable/disable running rviz in launcher -->
    <arg name="rviz" default="true" />
    <arg name="titania_ns" default="i3dr_titania" />

    <node pkg="halcon_match" type="matcher" name="halcon_matcher" output="screen" >
        <param name="reference_model" value="$(find halcon_match)/test/aluminium_tube.stl" />
        <remap from="/halcon_matcher/points2" to="$(arg titania_ns)/points2" />
        <param name="mesh_resource" value="$(find halcon_match)/test/aluminium_tube.dae" />
        <param name="confidence_threshold" value="0.2" />
        <param name="object_key" value="serfow_cylinder" />
    </node>

    <!-- DISPLAY -->
    <!-- run rviz with config file -->
    <node if="$(arg rviz)" pkg="rviz" type="rviz" name="rivz" 
        args="-d $(find halcon_match)/rviz/titania.rviz"/>

</launch>